DfE.FindInformationAcademiesTrusts/Pages/Trusts/Overview.cshtml

```diff
@@ -0,0 +1,96 @@
+﻿@page "{ukprn}"
+@model OverviewModel
+
+@{
+  Layout = "_TrustLayout";
+}
+
+<section>
+  <div class="govuk-!-margin-top-9 govuk-!-margin-bottom-1">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" data-testid="trust-summary">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="trust-information">Trust summary</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <dl class="govuk-summary-list">
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Total academies
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Academies in each local authority
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  <p class="govuk-!-margin-bottom-1">3 in North Lincolnshire</p>
+                  <p class="govuk-!-margin-bottom-1">1 in North East Lincolnshire</p>
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil numbers
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4,896
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil capacity <br>(% full)
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  6,463<br>(76%)
+                </dd>
+              </div>
+            </dl>
+          </div>
+        </section>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" data-testid="ofsted-ratings">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="ofsted-ratings">Ofsted ratings</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <table class="govuk-table" data-module="moj-sortable-table" aria-describedby="ofsted-ratings">
+              <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header" aria-sort="ascending">Rating</th>
+                  <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-third" aria-sort="none">Number of schools</th>
```

Comment:
the css for MOJ sortable table is manually setting a font-size using px, which won't change if the user changes font size. Can we create our own scss class, with an override for this?

e.g. 

```
.app-sortable-table {
   [aria-sort] button, [aria-sort] button:hover {
     font-size: {insert relevant govuk font mixin equivalent to 19px}
     }
}
```